SETUP/devex/README.md

```diff
@@ -0,0 +1,101 @@
+# DP Local development
+
+## docker-based
+
+The docker-based development environment runs MySQL and Apache/PHP in docker
+containers and bind mounts the repository. This allows you to make changes to
+the code directly on your computer and use a local web browser to interact with
+the code.
+
+The MySQL database and the code-generated files are stored in docker volumes.
+These will persist across container reboots.
+
+### Set up
+
+First, install either [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+-- free for personal use -- or [Rancher Desktop](https://rancherdesktop.io/).
+Note that if you use Rancher Desktop the commands are slightly different,
+consult their manual for details.
+
+_Note: All commands should be run relative to this directory._
```

Comment:
I guessed that "this directory" meant the one containing the README, and everything worked OK. It did cause me some concern in my naivety, that maybe it meant the directory at the top of this tree, i.e. SETUP's parent directory. This was because some later commands, such as `docker compose exec dp-dev SETUP/devex/dpdev-docker/init_dyndir.sh` appeared to be referencing files relative to that parent directory. All went well, but I don't know if the wording "this directory" could be adjusted slightly to ease the tension for novices such as me :)

Reply:
This is excellent feedback and something I pondered too. If we moved the `docker-compose.yml` file to the top-level directory (and updated the relative paths within it) the commands could be run from there instead which might feel more natural anyway.

Thank you very much for testing it out on Windows!